definitions/cloud_native_tech.md

```diff
@@ -5,9 +5,14 @@ category: concept
 ---
 ## Cloud Native Technology
 
-Cloud native technologies empower organizations to build and run scalable applications in modern, dynamic environments such as public, private, and hybrid clouds. They uphold the ‘promise of the cloud’ and leverage cloud computing benefits to their fullest. Designed from the ground up to exploit the capabilities of cloud computing, containers, service meshes, microservices, and immutable infrastructure exemplify this approach.
+### What it is
 
-There are often many downsides to traditional models of IT operation, some of which may be more prevalent than others. Traditional systems are: hard to scale; not fault-tolerant; not self-healing; inefficient due to poor utilisation of resources; the list goes on.
+Cloud native technologies, also referred to as the cloud native or new stack, are the technologies used to build [cloud native applications](https://github.com/cncf/glossary/blob/main/definitions/cloud_native_apps.md). Enabling organizations to build and run scalable applications in modern, dynamic environments such as public, private, and hybrid clouds, they uphold the ‘promise of the cloud’ and leverage cloud computing benefits to their fullest. They are designed from the ground up to exploit the capabilities of cloud computing and containers, service meshes, microservices, and immutable infrastructure exemplify this approach.
 
-Cloud native technologies enable loosely coupled systems that are resilient, manageable, and observable. Combined with robust automation, they allow engineers to make high-impact changes frequently and predictably with minimal toil. Desirable traits of cloud-native systems are easier to achieve with the cloud native stack.
+### The problem it addresses 
 
+The cloud native stack has many different technology categories, addressing a variety of challenges. If you have a look at the [CNCF Cloud Native Landscape](https://landscape.cncf.io/), you'll see how many different areas it touches upon. But on a high level, they address one main set of challenges: the downsides of traditional IT operations models, some of which may be more prevalent than others. Challenges include hard to scale, not fault-tolerant, not self-healing, inefficient due to poor utilization of resources, the list goes on.
```

Comment:
The last sentence is a little cumbersome due to the negative examples, and I'm sure there should be a "being" after "include". "The list goes on" strikes me as rather conversational for written documentation.

All good with the sentiment though.

Reply:
How does this sound: "Challenges include difficulties creating scalable, fault-tolerant, self-healing applications, as well as inefficient resource utilization, among others."